themes/nekonight/nekonight-base.sh

```diff
@@ -0,0 +1,35 @@
+function _omb_theme_nekonight_git_prompt_info() {
```

Comment:
Could you add `#! bash oh-my-bash.module` at the beginning of the file? You should check what other theme files do.